### ISSUE TITLE
Site Editor Navigation: Tooltip background contrast

### ISSUE THREAD

**Issue by jameskoster** [WordPress/gutenberg#26495]
Tooltips and the Navigation panel share a background color:

<img src="https://cldup.com/UmrMz6ql42-3000x3000.png" width="300">

I think it makes sense to update the Tooltip background color to [`$black`](https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_colors.scss#L6), at least in the Sidebar, but perhaps globally.

**Comment by mtias** [comment]
@ItsJonQ has also mentioned about having contextual dark mode for different parts of the UI. It could make sense for tooltips to be white within the context of the dark sidebar.
